FallProject/Controllers/ExpensesController.cs

```diff
@@ -36,11 +36,20 @@ public IActionResult List()
 
         [HttpPut]
         [Route("AddExpense")]
-        public IActionResult AddExpense(int saldo, int AccountId, string description)
+        public IActionResult AddExpense(int saldo, int AccountId, string description, string date)
```

Comment:
snyggt, ska ni lägga in en '<'input type="date"'>' i html formen ?

Reply:
Tack för påminnelsen, det ska vi såklart göra. Den försvann mellan stolarna igår.